migrations/r_12_05.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE academic_terms CHANGE term_id id BIGINT UNSIGNED;
+ALTER TABLE academic_terms CHANGE start_date date_start TIMESTAMP DEFAULT CURRENT_TIMESTAMP;
+ALTER TABLE academic_terms CHANGE end_date date_end TIMESTAMP DEFAULT CURRENT_TIMESTAMP;
+
+ALTER TABLE academic_terms ADD COLUMN canvas_id CHAR(255) AFTER id;
+
+ALTER TABLE course MODIFY term_id BIGINT UNSIGNED NOT NULL DEFAULT 1;
+
+UPDATE academic_terms SET id = 17700000000000108, canvas_id = "108" WHERE id = 1;
```

Comment:
Are those migrations/r_12_xx.sql script just for localhost deploy? I see the hardcoded canvas_id here.

Reply:
This conversion script is actually for production. These are the hard coded ids in our prod instance for academic terms. This conversion script *should* work fine with production. With dev we'll probably have to wipe out the data and start fresh. This academic_terms table will populate from canvas during the cron runs. I'll backup the database table before we roll these changes out though.